modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -472,6 +472,19 @@ export default function EditObjectFolder({
 						}}
 						objectDefinitionId={movedObjectDefinitionId}
 						objectFolders={objectFolders}
+						onAfterSubmit={() => {
```

Comment:
what do you think of changing the `onAfterSubmit` to `onAfterMoveObjectDefinition`? i did this already on my PR related the move action and it would be the same thing in the end; this name is follows a pattern that is present in some places of the code; but if you prefer the `onAfterSubmit`, we can change the other places to use it too. Which one do you think its better?

Reply:
Sure @rebsilva, I agree with this! I'll change now!